src/btrfs/fs.rs

```diff
@@ -96,7 +96,7 @@ enum FsInner {
 /// Holds a refcount on the FS while alive and decrements when struct is dropped (to prevent FS
 /// from being unmounted while we're debugging)
 pub struct Fs {
-    inner: FsInner,
+    pub inner: FsInner,
```

Comment:
Why make this field public? `FsInner` hides the details between a mounted and unmounted fs.